src/vm/ceeload.cpp

```diff
@@ -2577,7 +2577,7 @@ BOOL Module::IsNoStringInterning()
     return !!(m_dwPersistedFlags & NO_STRING_INTERNING);
 }
 
-BOOL Module::GetNeutralResourcesLanguage(LPCUTF8 * cultureName, ULONG * cultureNameLength, INT16 * fallbackLocation, BOOL cacheAttribute)
+BOOL Module::GetNeutralResourcesLanguage(LPCUTF8 * cultureName, ULONG * cultureNameLength, INT16 * fallbackLocation)
```

Comment:
This method, `m_pszCultureName`, `m_CultureNameLength`, `m_FallbackLocation` and `NEUTRAL_RESOURCES_LANGUAGE_IS_CACHED` can be all deleted.

Reply:
I kept `m_pszCultureName` as a flag when to fixup the string

Reply:
The calls to `StoreStructure` / `FixupPointerString` for `m_pszCultureName` can be deleted together with the field.

Reply:
Oh, the native `GetNeutralResourcesLanguage` can be removed completely?

Reply:
I think so